src/Tizen.Multimedia.Remoting/WebRTC/WebRTC.Events.cs

```diff
@@ -0,0 +1,157 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using static Interop;
+
+namespace Tizen.Multimedia.Remoting
+{
+    public partial class WebRTC
+    {
+        private NativeWebRTC.ErrorOccurredCallback _webRtcErrorCallback;
+        private NativeWebRTC.StateChangedCallback _webRtcStateChangedCallback;
+        private NativeWebRTC.NegotiationNeededCallback _webRtcNegotiationNeededCallback;
+        private NativeWebRTC.IceCandidateCallback _webRtcIceCandicateCallback;
+        private NativeWebRTC.TrackAddedCallback _webRtcTrackAddedCallback;
+        private NativeWebRTC.FrameEncodedCallback _webRtcFrameEncodedCallback;
+        private NativeDataChannel.CreatedCallback _webRtcDataChannelCreatedCallback;
+        private uint? _trackId;
+
+        public event EventHandler<WebRTCErrorOccurredEventArgs> ErrorOccurred;
+
+        public event EventHandler<WebRTCStateChangedEventArgs> StateChanged;
+
+        public event EventHandler<EventArgs> NegotiationNeeded;
+
+        public event EventHandler<WebRTCIceCandicateEventArgs> IceCandidate;
+
+        public event EventHandler<WebRTCTrackAddedEventArgs> TrackAdded;
+
+        public event EventHandler<WebRTCFrameEncodedEventArgs> AudioFrameEncoded;
+
+        public event EventHandler<WebRTCFrameEncodedEventArgs> VideoFrameEncoded;
+
+        public event EventHandler<WebRTCDataChannelCreatedEventArgs> DataChannelCreated;
```

Comment:
For this public event, we can rename it to 
WebRTCDataChannelEventArgs DataChannel (without 'Created').

The corresponding Web API of it is ondatachannel event handler of PeerConnection.
: https://developer.mozilla.org/en-US/docs/Web/API/RTCPeerConnection

Others seem fine..
: onnegotiationneeded -> NegotiationNeeded
: onicecandidate -> IceCandidate

Reply:
Done.